docs: Documented Offline page migration (#872)
front-commerce/developers.front-commerce.com: docs/03-extensions/theme-chocolatine/how-to/customize-the-offline-page.mdx

---
title: "Customize the offline page"
description: "This guide explains how to customize the offline page."
sidebar_position: 3
---

<p>{frontMatter.description}</p>

The offline page is accessible when you disable your network while navigating in

> your shop. If you have `FRONT_COMMERCE_ENV !== "production"` this page will also
> be available at `/__front-commerce/offline`. You can then edit the

Reviewer: **issue:** the page is always visible, even in production

**suggestion:** reword this part to be correct
```suggestion
your shop. You can preview it at `/__front-commerce/offline`. You can then edit the
```

Author: Indeed ! This dated from a previous iteration where the route was hidden in production mode. Will fix.
